workshop/README.md

```diff
@@ -1,9 +1,31 @@
-# Istio 101 workshop
+# Advanced Istio course (name TBD)
+[Istio](https://www.ibm.com/cloud/info/istio) is an open platform to connect, secure, and manage a network of microservices, also known as a service mesh, on cloud platforms such as Kubernetes in IBM Cloud Container Service. With Istio, manage network traffic, load balance across microservices, enforce access policies, verify service identity on the service mesh, and more.
 
-## Workshop Setup
-- [Exercise 1 - Accessing a Kubernetes cluster with IBM Cloud Container Service](exercise-1/README.md)
+In this course, you can see how to install Istio alongside microservices for a simple mock app called Guestbook. The microservices include a product web page, details, reviews, and ratings. When you deploy Guestbook's microservices into an IBM Cloud Container Service cluster where Istio is installed, you inject the Istio Envoy sidecar proxies in the pods of each microservice.
+
+**Note**: Some configurations and features of the Istio platform are still under development and are subject to change based on user feedback. Allow a few months for stablilization before you use Istio in production.
+
+## Objectives
+After you complete this course, you'll be able to: 
+- Download and install Istio in your cluster
+- Deploy the Guestbook sample app
+- Set up the Istio Ingress controller
+- Use metrics, logging and tracing to observe services
+- Perform simple traffic management, such as A/B tests and canary deployments
+- Secure your service mesh
+- Enforce policies for your microservices
+
+## Prerequisites
+You must create a [standard cluster in a Pay-As-You-Go and Subscription account](https://console.bluemix.net/registration/) to complete all the modules in this course. A free cluster in a Trial account does not have access to all the advanced features of this course. 
 
```

Comment:
I don't think this is entirely necessary as a pre-requisites.  User can do every thing in the trial account, the only thing is they need to access the istio-ingress via node-port and not do the front door portion.  cc @szihai

Reply:
When I saw "Ingress," I assumed Kube Ingress. OK, I'll update this.